implementations/rust/ockam/ockam_command/src/vault/util.rs

```diff
@@ -0,0 +1,32 @@
+use std::path::PathBuf;
+use std::sync::Arc;
+
+use anyhow::Ok;
+use ockam_api::config::cli;
+use ockam_vault::storage::FileStorage;
+use ockam_vault::Vault;
+use slug::slugify;
+
+pub fn check_if_vault_exists(path: PathBuf) -> bool {
+    path.as_path().exists()
+}
```

Comment:
This function is used once and it contains a single line, you should replace the function call with its contents.